SimPEG/simulation.py

```diff
@@ -41,10 +41,40 @@
 
 
 class BaseSimulation(props.HasModel):
+    r"""Base class for all geophysical forward simulations in SimPEG.
+
+    The ``BaseSimulation`` class defines properties and methods inherited by
+    practical simulation classes in SimPEG.
+
+    .. important::
+        This class is not meant to be instantiated. You should inherit from it to
+        create your own simulation class.
+
+    Parameters
+    ----------
+    mesh : discretize.base.BaseMesh, optional
+        Mesh on which the forward problem is discretized. This is not necessarily
+        the same as the mesh on which the simulation is defined.
```

Comment:
```suggestion
        Mesh on which the forward problem is discretized.
```

The mesh that is provided should be used by the forward simulation, so I think this comment should be removed